src/objects/Package_Settings__c.object

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <customSettingsType>Hierarchy</customSettingsType>
+    <enableFeeds>false</enableFeeds>
+    <label>NPSP Package Settings</label>
+    <visibility>Protected</visibility>
+    <fields>
+        <fullName>Namespace_Prefix__c</fullName>
+        <description>The active package namespace prefix for NPSP.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>Do not edit.</inlineHelpText>
```

Comment:
Though this is technically only visible to a User in an unmanaged package state, I think the InlineHelpText should be "The active package namespace prefix for NPSP. Do not edit manually"